### PR TITLE
feat: resistor color code experiment

### PR DESCRIPTION
**Pull request by saidabdul80**

### CODE REVIEW

File: resources/js/components/ExpRightNav.vue [hunk 1]
```diff
@@ -3,17 +3,20 @@
 		<div class="containerR" id="tools" style="height:100%;">	
 			
 		   	<div class="input-alt"></div>	
-		   	<span v-if="vicelab==1" class="pr z-1" >		 
+		   	<span v-if="vicelab==true" class="pr z-1" >
```
Comment (Auwalms) [inline resources/js/components/ExpRightNav.vue]: You too like vicelab this guy,  why not use a descriptive variable name instead? Please correct. @saidabdul80

File: resources/js/components/ExpRightNav.vue [hunk 2]
```diff
@@ -116,12 +120,15 @@
 		},
 
         props:{
-            toolsizes: String,
+            toolsizes: Array,
             url:String,
-            toolState:Boolean,
-            vicelab:String
+            toolstate:Boolean,
+            vicelab:Boolean,
```
Comment (Auwalms) [inline resources/js/components/ExpRightNav.vue]: Remember to update this as well @saidabdul80

**Review by Auwalms — COMMENTED** [review]
Please update the Vicelab mentions.

**Comment by saidabdul80** [comment]
Ok sir

File: resources/js/components/ExpRightNav.vue (hunk 1 above)
Comment (abdullahikawu) [reply resources/js/components/ExpRightNav.vue]: please correct accordingly

File: resources/js/components/ExpRightNav.vue [hunk 3]
```diff
@@ -3,17 +3,20 @@
 		<div class="containerR" id="tools" style="height:100%;">	
 			
 		   	<div class="input-alt"></div>	
-		   	<span v-if="vicelab==1" class="pr z-1" >		 
+		   	<span v-if="vicelab==true" class="pr z-1" >		 
 		   	 	<vicelabtools has='1'  ></vicelabtools>
```
Comment (abdullahikawu) [inline resources/js/components/ExpRightNav.vue]: same here

File: resources/js/components/ExpRightNav.vue [hunk 4]
```diff
@@ -34,20 +37,21 @@
...
-            <div v-if="toolState==true" v-for="tool in toolSizes"  :key="tool" @click="addactivate;changeApparatus(tool)"  v-bind:style="{width:tool+'px'}" class="box">
+            <div v-if="toolstate==true" v-for="tool in toolsizes"  :key="tool" @click="addactivate;changeApparatus(tool)"  v-bind:style="{width:tool+'px'}" class="box">
             		Size
         	</div> 			                
-		</div>
+		</div> -->
 	</div>
 </template>
 <script>
 	import vicelabtools from './vicelabExpEquip';
```
Comment (abdullahikawu) [inline resources/js/components/ExpRightNav.vue]: I think you can rename the package here too. From vicelabtools to vlabtools or experimenttools, or just something descriptive. Also for the class there

File: resources/js/components/ExpRightNav.vue [hunk 5]
```diff
@@ -34,20 +37,21 @@
...
+	import electricity from './electricityEquipment';
```
Comment (abdullahikawu) [inline resources/js/components/ExpRightNav.vue:50]: excellent, this is exactly what we mean. So, what's stopping you from renaming the previous code block like this ?

File: resources/js/components/ExpRightNav.vue [hunk 6]
```diff
@@ -34,20 +37,21 @@
...
 	export default{
 	components:{
 		vicelabtools,
```
Comment (abdullahikawu) [inline resources/js/components/ExpRightNav.vue]: rename accordingly please

File: resources/js/components/ExpRightNav.vue (hunk 2 above)
Comment (abdullahikawu) [reply resources/js/components/ExpRightNav.vue]: sure

File: resources/views/experiment/photovoltaic2.blade.php [hunk 7]
```diff
@@ -39,8 +39,9 @@
    			    </div>
    			    <!-- end experiment -->
 
-            <div  class="zero-space vh-2 exprightNav" id="rightNav">               
-               <v-rightnav toolsizes="false" url="{{route('photovoltaic2')}}" vicelab=1 ></v-rightnav>
+            <div  class="zero-space vh-2 exprightNav" id="rightNav">                             
+               <v-rightnav class="vh-2" :toolsizes=false :vicelab=true   url="{{route('photovoltaic2')}}" ></v-rightnav>
```
Comment (abdullahikawu) [inline resources/views/experiment/photovoltaic2.blade.php]: this too, please - vicelab and photovoltaic

File: resources/views/experiment/photovoltaic1.blade.php [hunk 8]
```diff
@@ -39,8 +39,8 @@
    			    </div>
    			    <!-- end experiment -->
 
-            <div  class="zero-space exprightNav" id="rightNav">               
-               <v-rightnav class="vh-2" toolsizes="false" url="{{route('photovoltaic1')}}" vicelab=1 ></v-rightnav>
+            <div  class="zero-space exprightNav" id="rightNav">       
+               <v-rightnav class="vh-2" :toolsizes=false :vicelab=true   url="{{route('photovoltaic1')}}" ></v-rightnav>
```
Comment (abdullahikawu) [inline resources/views/experiment/photovoltaic1.blade.php]: vicelab? photovoltaic???!

**Review by abdullahikawu — CHANGES_REQUESTED** [review]
please effect the changes as observed and resend another PR with the changes

**Review by Auwalms — APPROVED** [review]
LGTM

File: public/css/style.css [hunk 9]
```diff
@@ -147,6 +147,12 @@ main {
...
   overflow-y: scroll;
 }
+.p-display-none{
+  display: none;
+}
+.p-display-flex{
+  display: flex;
+}
 .p-danger {
   background: #ff0000 !important;
```
Comment (abdullahikawu) [inline public/css/style.css:157]: !important?

File: public/css/style.css [hunk 10]
```diff
@@ -176,6 +204,9 @@ main {
 .h1 {
   height: 26px;
 }
+.h2{
+  height: 50px !important;
```
Comment (abdullahikawu) [inline public/css/style.css:208]: please can we avoid the use of !important?

File: public/css/style.css [hunk 11]
```diff
@@ -291,18 +322,28 @@ main {
...
 .fs1 {
-  font-size: 0.9em;
+  font-size: 0.9em !important;
 }
 .fs8 {
   font-size: 1.5em !important;
 }
 .fs01 {
-  font-size: 0.8em;
+  font-size: 0.8em !important;
```
Comment (abdullahikawu) [inline public/css/style.css:331]: this might have work without adding !important? So, why force important on it?

File: public/css/style.css [hunk 12]
```diff
@@ -291,18 +322,28 @@ main {
...
 }
 .fs9 {
-  font-size: 2em;
```
Comment (abdullahikawu) [inline public/css/style.css:303]: please retain this without !important

File: resources/js/app.js [hunk 13]
```diff
@@ -12,15 +14,21 @@ import Vuex from 'vuex'
...
+import storedata from "./store/exp1";
+//import expvicelab3 from "./store/exp2";
 /*
 const store = new Vuex.Store(
    storeData
 )*/
-const store = new Vuex.Store(   expvicelab2 )
-const store2 = new Vuex.Store(   expvicelab3 )
+const store = new Vuex.Store(   storedata )
+/*const store2 = new Vuex.Store(   expvicelab3 )*/
```
Comment (abdullahikawu) [inline resources/js/app.js:30]: vicelab?

Comment (abdullahikawu) [inline resources/js/app.js:30]: remove comment

File: resources/views/experiment/circuitconnection1.blade.php [hunk 14]
```diff
@@ -34,13 +36,13 @@
...
-               <v-photovoltaic1 style="display: none;" id="experimentSheet"></v-photovoltaic1>
+               <v-circuitconnection1 style="display: none;" id="experimentSheet"></v-circuitconnection1>
                <v-start hourdata="1" munitedata="30" starteddata="{{$started}}" ></v-start>               
    			    </div>
    			    <!-- end experiment -->
 
-            <div  class="zero-space exprightNav" id="rightNav">               
-               <v-rightnav class="vh-2" toolsizes="false" url="{{route('photovoltaic1')}}" vicelab=1 ></v-rightnav>
+            <div  class="zero-space exprightNav" id="rightNav">       
+               <v-rightnav class="vh-2" :toolsizes=false :vicelab=true   url="{{route('circuitconnection1')}}" ></v-rightnav>
```
Comment (abdullahikawu) [inline resources/views/experiment/circuitconnection1.blade.php]: you forgot to replace vicelab here

**Review by abdullahikawu — APPROVED** [review]
well done for replacing vicelab in the codebase as requested, just a few more that you missed. Two other things to kindly note, first, kindly stop using important as much as possible. Also, leaving the comments within the codebase come is not the best practice. Thank you.

**Pull request merged**